modules/network-observability-quickfilter.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// network_observability/observing-network-traffic.adoc
+:_content-type: REFERENCE
+[id="network-observability-quickfilter{context}"]
+= Filtering the network traffic
+By default, the Network Traffic page displays the traffic flow data in the cluster based on the default filters configured in the FlowCollector instance. You can use the filter options to observe the required data by changing the preset filter. Following are the available filter options:
+
+
+.Quick filters
+The default values in *Quick filters* drop-down are defined in the `FlowCollector` config. You can modify the options from console. 
+
+.Advanced filters
+You can set the advanced filters by providing the parameter to be filtered and its corresponding text value. The section *Common* in the parameter drop-down list filters the results that match either *Source* or *Destination*. To enable or disable the applied filter, you can click on the applied filter listed below the filter options.
```

Comment:
Suggestion: you might like using a [description list](https://docs.asciidoctor.org/asciidoc/latest/lists/description/#basic-description-list) here
```suggestion
Quick filters:: The default values in *Quick filters* drop-down are defined in the `FlowCollector` config. You can modify the options from console. 

Advanced filters:: You can set the advanced filters by providing the parameter to be filtered and its corresponding text value. The section *Common* in the parameter drop-down list filters the results that match either *Source* or *Destination*. To enable or disable the applied filter, you can click on the applied filter listed below the filter options.
```

Reply:
aah yes! That fits in better. Thanks for the suggestion!